README.md

```diff
@@ -33,16 +33,13 @@ To further configure your plugin, use the following custom parameters in your `s
 
 | Parameter                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `flushMetricsToLogs`        | Send custom metrics by using logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter, it's required to set `apiKey` or if encrypted, `apiKMSKey` or `apiKeySecretArn`. `flushMetricsToLogs` is ignored when `addExtension` is true.                                                                                                                                             |
+| `flushMetricsToLogs`        | Send custom metrics by using logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter, it's required to set `DATADOG_API_KEY` in your environment, or if encrypted, `apiKMSKey` or `apiKeySecretArn` in the configuration. `flushMetricsToLogs` is ignored when `addExtension` is true.                                                                                          |
 | `site`                      | Set which Datadog site to send data, this is only used when `flushMetricsToLogs` is `false` or `addExtension` is `true`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com`, `us5.datadoghq.com`, and `ddog-gov.com`. The default is `datadoghq.com`.                                                                                                                                                              |
-| `apiKey`                    | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `addExtension` is `true`. Defining `apiKey` will add the Datadog API key directly to your Lambda functions as an environment variable. For more information about getting a Datadog API key, see the [API key documentation][3].                                                                                                                                    |
```

Comment:
Like we talked about in person, I think it's a bad idea to delete these keys. People often write their serverless.yaml files to reference secrets using different mechanisms, (eg, it's possible to read values from KMS, or as a command line argument), and need to be able to set the apiKey/appKey as a property inside the template, not just as an environment variable. We should warn people not to commit the keys in plaintext if they do use these field.

Reply:
I agree with not deleting the keys from the docs. We could add a warning/deprecated message just like Darcy suggested. Maybe in the note?

Reply:
Yep, on it!